### PR TITLE
samples: peripheral: radio_test: Add nRF7002 DK support

### PR DESCRIPTION
**Pull request by krish2718**
Add nRF7002 DK support for radio test and empty application core samples, this is same as nRF5340 as there are no Wi-Fi changes but helps use test nRF7002 definitions (DTS).

Signed-off-by: Krishna T <krishna.t@nordicsemi.no>

### CODE REVIEW

File: samples/peripheral/radio_test/README.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ The sample supports the following development kits:
 You can use any one of the development kits listed above.
 
 .. note::
-   On nRF5340 DK, the sample is designed to run on the network core.
+   On nRF5340 DK and nrf7002 DK, the sample is designed to run on the network core.
```
Comment (mjd90) [inline samples/peripheral/radio_test/README.rst:27]:
```suggestion
   On nRF5340 DK and nRF7002 DK, the sample is designed to run on the network core.
```

**Review by krga2022 — APPROVED** [review]
Ble_radio test between two boards is working as expected
Transmitted 10000 packets on TX and able to receive all 10000 packets on RX board.

**Review by mjd90 — APPROVED** [review]
Looks good.

File: samples/peripheral/radio_test/README.rst [hunk 2]
```diff
@@ -160,6 +160,9 @@ You can use the following command:
 
   west build -b nrf5340dk_nrf5340_cpunet -- -DCONF_FILE=prj_ipc_shell.conf -DSHIELD=nrf21540_ek -Dremote_shell_SHIELD=nrf21540_ek
 
+.. note::
+    Above examples are applicable for |nRF7002DKnoref| as well, please use the board name nrf7002dk_nrf5340_cpunet in the commands.
```
Comment (richabp) [inline samples/peripheral/radio_test/README.rst:164]:
```suggestion
    You can also build the sample with the remote IPC Service Shell for the |nRF7002DKnoref| using the ``nrf7002dk_nrf5340_cpunet`` build target in the commands.
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by richabp — APPROVED** [review]
